src/main/java/org/symphonyoss/symphony/messageml/elements/PersonSelector.java

```diff
@@ -70,11 +76,12 @@ public void validate() throws InvalidInputException {
       assertAttributeValue(REQUIRED_ATTR, VALID_VALUES_FOR_REQUIRED_ATTR);
     }
 
-    if(getAttribute(SELECTED_USERS_IDS) != null){
-      String selectedUsersIds = getAttribute(SELECTED_USERS_IDS);
-      if(!selectedUsersIds.matches(SELECTED_USERS_IDS_FORMAT)){
+    if(getAttribute(VALUE_ATTR) != null){
+      try{
```

Comment:
formatting seems a bit of (missing spaces)

Reply:
yes you are right, I fixed the formatting